cpp/src/lance/format/schema.h

```diff
@@ -232,6 +232,7 @@ class Field final {
   friend class FieldVisitor;
   friend class ToArrowVisitor;
   friend class WriteDictionaryVisitor;
+  friend class LoadDictionaryVisitor;
```

Comment:
What's the difference between Read and Load?

Reply:
Oh, this is loading the string values of the dictionary. I am fine to rename it.  wdyt.

Reply:
Let's just be consistent if it's doing the same thing

Reply:
fixed.